basic-user: Tries to write to FIFO
libarchive: error: Writing content object: No such file or directory
test-symbols: requires elfutils utilities, we only compile libraries
pull-mirrorlist: unknown 
fsck-collections: unknown
basic-user-only: Fails on CI, unknown
pull-bare: Fails on CI on 64bit arches, unknown
refs: fails on CI on 64bit arches, unknown
concurrency: fails on CI armhf, unknown

diff --git a/Makefile-tests.am b/Makefile-tests.am
index 417a304..4fa15ea 100644
--- a/Makefile-tests.am
+++ b/Makefile-tests.am
@@ -46,7 +46,7 @@ endif
 uninstalled_test_data = tests/ostree-symlink-stamp tests/ostree-prepare-root-symlink-stamp \
 			tests/ostree-remount-symlink-stamp
 
-dist_uninstalled_test_scripts = tests/test-symbols.sh tests/coccinelle.sh
+dist_uninstalled_test_scripts = tests/coccinelle.sh
 
 # This logic implements ENABLE_INSTALLED_TESTS_EXCLUSIVE; see below.
 # The goal here if installed tests are enabled, we explicitly make the
@@ -61,8 +61,6 @@ test_programs = \
 	$(NULL)
 _installed_or_uninstalled_test_scripts = \
 	tests/test-basic.sh \
-	tests/test-basic-user.sh \
-	tests/test-basic-user-only.sh \
 	tests/test-basic-root.sh \
 	tests/test-pull-subpath.sh \
 	tests/test-archivez.sh \
@@ -72,9 +70,7 @@ _installed_or_uninstalled_test_scripts = \
 	tests/test-commit-sign.sh \
 	tests/test-export.sh \
 	tests/test-help.sh \
-	tests/test-libarchive.sh \
 	tests/test-parent.sh \
-	tests/test-pull-bare.sh \
 	tests/test-pull-bareuser.sh \
 	tests/test-pull-bareuseronly.sh \
 	tests/test-pull2-bareuseronly.sh \
@@ -116,18 +112,14 @@ _installed_or_uninstalled_test_scripts = \
 	tests/test-xattrs.sh \
 	tests/test-auto-summary.sh \
 	tests/test-prune.sh \
-	tests/test-concurrency.py \
-	tests/test-refs.sh \
 	tests/test-demo-buildsystem.sh \
 	tests/test-switchroot.sh \
 	tests/test-pull-contenturl.sh \
-	tests/test-pull-mirrorlist.sh \
 	tests/test-summary-update.sh \
 	tests/test-summary-view.sh \
 	tests/test-no-initramfs.sh \
 	tests/test-create-usb.sh \
 	tests/test-find-remotes.sh \
-	tests/test-fsck-collections.sh \
 	tests/test-init-collections.sh \
 	tests/test-prune-collections.sh \
 	tests/test-refs-collections.sh \

